### PR TITLE
Adding tests and prettier

### PR DESCRIPTION
**Pull request by SharonStrats**
Added more tests and fixed a few.

### CODE REVIEW

**Comment by michielbdejong** [comment]
Can you remove TestTracking.xlsx from the repo root?

File: test/unit/widgets/forms.test.ts [hunk 1]
```diff
@@ -24,11 +23,9 @@ import {
...
   sortBySequence
 } from '../../../src/widgets/forms'
-import * as ns from '../../../src/ns'
-jest.mock('rdflib')
-jest.mock('solid-auth-client')
-const dom = new JSDOM('<!DOCTYPE html><p>Hello world</p>').window.document
-const element = dom.createElement('div')
+
+// jest.mock('rdflib')
+// jest.mock('solid-auth-client')
```
Comment (michielbdejong) [inline test/unit/widgets/forms.test.ts]: Why comment these out?

File: test/unit/tabs.test.ts [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
+const tabs = require('../../src/tabs')
+const RdfLib = require('rdflib')
 
 describe('tabWidget', () => {
   it('exists', () => {
     expect(tabs.tabWidget).toBeInstanceOf(Function)
   })
-  /* TODO: check this because it says kb.the is not a function
-            this is within the tabs.tabWidget function..
+  /*
```
Comment (michielbdejong) [inline test/unit/tabs.test.ts]: Use `it.skip` instead of commenting it out

File: test/unit/table.test.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+const window = new JSDOM('<!DOCTYPE html><p>Hello world</p>').window
+const dom = window.document
+
+describe('renderTableViewPane', () => {
+  it('exists', () => {
+    expect(renderTableViewPane).toBeInstanceOf(Function)
+  })
+  /*
+  it('runs', () => {
```
Comment (michielbdejong) [inline test/unit/table.test.ts]: skip instead of comment

File: test/unit/pad.test.ts [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+describe('lightColorHash', () => {
+  it('exists', () => {
+    expect((pad as any).lightColorHash).toBeInstanceOf(Function)
+  })
+  it('returns #ffffff when an author is not provided', () => {
+    expect((pad as any).lightColorHash(null)).toBe('#ffffff')
+  })
+  it('returns a value when given an author', () => {
+    const author = { uri: 'https://sharonstrats.inrupt.net/profile/card#me' }
+    expect((pad as any).lightColorHash(author)).toBe('#dac2dc')
```
Comment (michielbdejong) [inline test/unit/pad.test.ts]: Can you add a comment explaining why you expect this?
E.g.
```js
// '#dac2dc' is the hash of 'https://sharonstrats.inrupt.net/profile/card#me',
// see (source code file name), line (line number).

File: test/unit/widgets/forms.test.ts (hunk 1 above)
Comment (SharonStrats) [reply test/unit/widgets/forms.test.ts]: I wasn't sure I needed them.  It works without, however I may need to add them back in as I keep adding tests.

**Pull request merged**